charts/spinnaker/README.md

```diff
@@ -54,9 +54,10 @@ kubeConfig:
   enabled: true
   secretName: my-kubeconfig
   secretKey: config
-  contexts:
-  # Names of contexts available in the uploaded kubeconfig
-  - production
+  accounts:
+    production:
+       # Context available in the uploaded kubeconfig
+       context: production
```

Comment:
Should we also mention the other supported fields on this object? I.e. `namespaces`, `{read,write}Permissions`.